db/memtable.cc

```diff
@@ -534,6 +534,10 @@ struct Saver {
 };
 }  // namespace
 
+//static void UnrefMemTable(void* s, void*) {
+// reinterpret_cast<MemTable*>(s)->Unref();
+//}
```

Comment:
sure. i wanted to make it easier for you to see which options we took into account before resorting to string copy for memtables.